README.md

```diff
@@ -1,19 +1,57 @@
-AMX Mod X
-=========
 
-AMX Mod X - Half-Life 1 Scripting and Server Administration
+<p align="center">
+ <img src="https://github.com/alliedmodders/amxmodx/blob/master/editor/studio/AMXXLarge.bmp"/>
+</p>
+
+**AMX Mod X** is a [Metamod](https://github.com/jkivilin/metamod-p) plugin for [Half-Life 1](https://github.com/ValveSoftware/halflife). It provides comprehensive scripting for the game engine and its mods. Scripts can intercept network messages, log events, commands, client commands, set cvars, modify entities, and more. AMX Mod X also has a system for extending native scripting through modules, leading to outside support for things like MySQL and Sockets.
+
+Feature list
```

Comment:
I would remove the feature list. It's outdated and unnecessary. Keeping just the first paragraph seems fine.